packages/typespec-client-generator-core/test/types/general-decorators-list.test.ts

```diff
@@ -327,4 +327,29 @@ describe("typespec-client-generator-core: general decorators list", () => {
       ]);
     });
   });
+
+  describe("csharp only decorator", () => {
```

Comment:
Is it on purpose that there is no actual accessor at all?

Reply:
Not quite follow here, what do you mean by "no actual accessor"?

Reply:
There is no api to get the data apart from the generic decorator collector

Reply:
Right. the only usage  I can see is to flow into generic decorator list as a flag for the generator.